changelog/44.breaking.md

```diff
@@ -0,0 +1 @@
+Moved `openscm_units._unit_registry` to `openscm_units.registry` i.e. into the public API
```

Comment:
Is that really a breaking change? 

Why move it into the public API? `ScmUnitReigstry` and `unit_registry` are imported in `openscm_units/__init__.py`

Reply:
> Is that really a breaking change?

Ye maybe not. I can change to feature or trivial, let me know what you'd prefer

> Why move it into the public API? ScmUnitReigstry and unit_registry are imported in openscm_units/__init__.py

Haha you caught me. I couldn't work out how to get the docs to build properly so I went the lazy option :) Maybe I need to put them in `__all__` and then it would work. I'm happy to revert this and have the docs just be super thin instead, doesn't really matter.

Reply:
@lewisjared decision/thoughts ?